src/config_default_credentials.rs

```diff
@@ -103,5 +106,31 @@ struct RefreshRequest<'a> {
     refresh_token: &'a str,
 }
 
+#[cfg(any(target_os = "linux", target_os = "macos"))]
+fn get_user_credentials_path() -> Result<PathBuf, Error> {
```

Comment:
We should avoid `get_()` prefixes per the API naming guidelines. Also I think it would be nice to deduplicate the part where we push the `USER_CREDENTIALS_PATH`, so how about we name this `config_dir()`?

Reply:
Renamed to `config_dir()` and extracted the `USER_CREDENTIALS_PATH` bit.